spec/features/admin/protocols/index_spec.rb

```diff
@@ -47,5 +48,16 @@
         protocol_page.click_on_message_close_button
       end
     end
+
+    it "verify manage section overlay for manage_protocols permission" do
```

Comment:
If I understand correctly, this PR introduces several specs that test which tabs are visible and not-visible in the "Manage" section for different permissions. Based on how you'd like to address my first PR comment, could we make all of these test descriptions consistent?